src/desktop/components/main_layout/templates/instant-page.jade

```diff
@@ -1,4 +1,4 @@
-if sd.ENABLE_INSTANT_PAGE
+if sd.INSTANT_PAGE_AB_TEST === 'control'
```

Comment:
@anipetrov - confirming this is what ya'll would like: since it's been on, it's the control, correct?